### ISSUE TITLE
glib 2.0 dependency error when installing the development branch

### ISSUE THREAD

**Issue by dasl-** [mikebrady/shairport-sync#1426]
Hi there, today I tried installing the development branch on a fresh install of the latest version of raspbian bullseye. I got an error building shairport-sync:

```
...
checking for alsa... yes
checking for gio-unix-2.0 >= 2.30.0... no
configure: error: the glib 2.0 library is required -- libglib2.0-dev suggested!
```

I fixed this by running:
```
sudo apt -y install --no-install-recommends libglib2.0-dev
```

Perhaps the build instructions need to be updated. Maybe something recent changed? https://github.com/mikebrady/shairport-sync/blob/development/BUILDFORAP2.md#tools-and-libraries

**Comment by mikebrady** [comment]
Thanks. Can you give the Shairport Sync configuration string please? You can get this by executing:
```
$ shairport-sync -V
```

**Comment by dasl-** [comment]
Here's the output:
```
% shairport-sync -V
4.1-dev-189-g8b483c09-AirPlay2-OpenSSL-Avahi-ALSA-soxr-metadata-dbus-sysconfdir:/etc
```

**Comment by mikebrady** [comment]
Yes, that's what it is -- if you need the D-Bus interface, then you need to add the `glib` library.

But, you are actually correct -- previously the library was included in the list of libraries to be loaded. It was removed recently for the basic case since it's not needed for that.

Thanks for reporting the issue and helping to improve Shairport Sync!
